docs/docusaurus.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  onBrokenLinks: 'warn',
+  onBrokenLinks: 'error',
```

Comment:
This line probably can be removed. I think I added it because when we upgraded docusaurus versions, they changed the defaults and I didn't want to be blocked on fixing broken links. If all the links have been fixed, removing the line should return to the default behavior of `error`.

Reply:
Don't we also want to be protected against changes of defaults on Docusaurus' side? If this is a feature we explicitly want to rely on, perhaps it's useful to state it as such? (No strong feelings of course…)

Reply:
That sounds fine.